### PR TITLE
init push for adding changie to redshift

### LINKED ISSUE #151: [CT-1011] Add changie to dbt-redshift
### Describe the feature
implement `changie` in all adapters to reach parity with `dbt-core`

### Who will this benefit?
Users: will have easy format to verify they have created a changelog for their pr, and simple interactive mode to create said changelog.

Maintainers: will be able to easily batch PR's by release/ pre-release/ easier release system.

### Are you interested in contributing this feature?
Yes

### PR DESCRIPTION
**Pull request by McKnight-42**
resolves #151 

### Description

Adding `changie` to all adapter repos to match parity with `dbt-core`.

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-redshift next" section.

### CODE REVIEW

**Review by emmyoop — APPROVED** [review]
🚀

File: .changie.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          {{- $contributorDict := set $contributorDict $author $prList }}
+        {{- end }}
+      {{- end}}
+    {{- end}}
+  {{- end }}
+  {{- /* no indentation here for formatting so the final markdown doesn't have unneeded indentations */}}
+  {{- if $contributorDict}}
+  ### Contributors
+  {{- range $k,$v := $contributorDict }}
+  - [@{{$k}}](https://github.com/{{$k}}) ({{ range $index, $element := $v }}{{if $index}}, {{end}}[#{{$element}}](https://github.com/dbt-labs/dbt-redshift/pull/{{$element}}){{end}})
```
Comment (emmyoop) [inline .changie.yaml:60]:
```suggestion
### Contributors
{{- range $k,$v := $contributorDict }}
- [@{{$k}}](https://github.com/{{$k}}) ({{ range $index, $element := $v }}{{if $index}}, {{end}}[#{{$element}}](https://github.com/dbt-labs/dbt-redshift/pull/{{$element}}){{end}})
```

Can't have these spaces.  They carry over.

**Review by emmyoop — APPROVED** [review]
Tiny format issue.  Then good to go!

File: .changie.yaml (hunk 1 above)
Comment (McKnight-42) [reply .changie.yaml:60]: believe that should be fixed now.

**Pull request merged**